internal/distro/fedora/distro.go

```diff
@@ -536,6 +538,24 @@ func (t *imageType) PackageSets(bp blueprint.Blueprint, options distro.ImageOpti
 		}
 	}
 
+	// In case of Cloud API, this method is called before the ostree commit
+	// resolved. Unfortunately, initializeManifest when called for an ostree
+	// installer returns an error.
+	//
+	// Work around this by providing a dummy FetchChecksum to convince the
+	// method that it's fine to initialize the manifest. Note that the ostree
+	// content has no effect on the package sets, so this is fine.
+	//
+	// See: https://github.com/osbuild/osbuild-composer/issues/3125
+	//
+	// TODO: Remove me when it's possible to initialize the get the package
+	//       set chain without resolving he ostree reference before. Also
+	//       remove the test for this workaround
```

Comment:
typo: `possible to get the package set chain`

Reply:
fixed, thanks!